cf_remote/spawn.py

```diff
@@ -307,9 +308,12 @@ def spawn_vm_in_aws(
     size=None,
     role=None,
 ):
-    driver = get_cloud_driver(Providers.AWS, aws_creds, region)
-    existing_vms = driver.list_nodes()
-
+    try:
+        driver = get_cloud_driver(Providers.AWS, aws_creds, region)
+        existing_vms = driver.list_nodes()
+    except InvalidCredsError as error:
+        print("\nInvalid credentials, check cloud_config.json file, details: \n" + str(error))
+        return None
```

Comment:
Same as in other PR. This function already uses `raise ValueError` to handle erors. It seems to me you should also use that here, unless you have a good reason not to.

Reply:
no; discussed the better way to do this in person.